src/features/content-explorer/content-explorer/ContentExplorer.scss

```diff
@@ -72,22 +72,56 @@
         }
     }
 
+    .bdl-ContentExplorer-subheader {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+
+        .checkbox-container,
+        .toggle-container {
+            margin: 0;
+        }
+
+        .bdl-ContentExplorerSelectAll-checkbox {
+            margin-right: 4.45 * $bdl-grid-unit;
+        }
+    }
+
     .content-explorer-select-all-container {
-        display: block;
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        width: 100%;
         height: 8 * $bdl-grid-unit;
+        margin-top: -5px;
+        margin-bottom: 5px;
 
         .content-explorer-select-all-items-counter {
             margin-left: 2px;
         }
 
         .content-explorer-select-all-checkbox-label {
-            float: right;
-            margin-right: 42px;
+            margin-right: 10px;
+            margin-left: auto;
+        }
+    }
+
+    .bdl-ContentExplorerIncludeSubfolders {
+        display: flex;
+        align-items: center;
+        height: 10 * $bdl-grid-unit;
+
+        .bdl-ContentExplorerIncludeSubfolders-toggleIconContainer {
+            display: flex;
+            align-items: center;
+        }
+
+        .bdl-ContentExplorerIncludeSubfolders-icon {
+            margin-left: 120px;
```

Comment:
what is this margin for? shouldn't it be on the toggle or the outer div?

Reply:
It is because the info badge icon is not a part of the label of the toggle component, so for now it needs the left margin to be positioned correctly with the toggle

Reply:
okay I see what's happening here now. we can't have this fixed margin because it won't work for different languages. i.e. the text would be different lengths so we can't rely on a fixed margin

we'll have to put the icon into the label like this: https://github.com/box/box-ui-elements/blob/master/src/features/advanced-content-insights/AdvancedContentInsightsToggle.tsx#L33-L55